static/sass/style.css

```diff
@@ -562,7 +563,7 @@ sub {
   bottom: -0.25em; }
 
 pre, code, kbd, samp, var {
-  font-family: Consolas, "Lucida Console", "Liberation Mono", "DejaVu Sans Mono", "Bitstream Vera Sans Mono", "Courier New", monospace, sans-serif; }
+  font-family: Consolas, 'Lucida Console','Liberation Mono', 'DejaVu Sans Mono', 'Bitstream Vera Sans Mono', 'Courier New', monospace, sans-serif; }
```

Comment:
Is it possible that someone updated it [here](https://github.com/python/pythondotorg/blob/master/static/sass/_functions.scss#L351) and didn't re-generate the CSS files? 

Looks like it might've crept in [here](https://github.com/python/pythondotorg/pull/33/files#diff-13d47ca5d9f026b9ccaf91f97b4d8438R259)